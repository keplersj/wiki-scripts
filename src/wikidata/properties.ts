// See [software version identifier](https://www.wikidata.org/wiki/Property:P348) on Wikidata
export const softwareVersionIdentifier = "P348";

// See [publication date](https://www.wikidata.org/wiki/Property:P577) on Wikidata
export const publicationDate = "P577";

// See [official website](https://www.wikidata.org/wiki/Property:P856) on Wikidata
export const officialWebsite = "P856";

// See [bug tracker system](https://www.wikidata.org/wiki/Property:P1401) on Wikidata
export const bugTrackerSystem = "P1401";

// See [reference URL](https://www.wikidata.org/wiki/Property:P854) on Wikidata
export const sourceReferenceUrl = "S854";

// See [retrieved](https://www.wikidata.org/wiki/Property:P813) on Wikidata
export const sourceRetrieved = "S813";

// See [npm package](https://www.wikidata.org/wiki/Property:P8262) on Wikidata
export const npmPackage = "P8262";

// See [crates.io ID](https://www.wikidata.org/wiki/Property:P4763) on Wikidata
export const cratesIoId = "P4763";

// See [RubyGems gem](https://www.wikidata.org/wiki/Property:P5566) on Wikidata
export const rubygemsGem = "P5566";

// See [Repology project name](https://www.wikidata.org/wiki/Property:P6931) on Wikidata
export const repologyProjectName = "P6931";

// Se [Flathub ID](https://www.wikidata.com/wiki/Property:P4655) on Wikidata
export const flathubId = "P4655";
