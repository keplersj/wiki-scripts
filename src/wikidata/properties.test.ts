import * as properties from "./properties.js";

describe("Wikidata Properties Aliases", () => {
  it("exports as expected", () => {
    expect(properties).toMatchSnapshot();
  });
});
