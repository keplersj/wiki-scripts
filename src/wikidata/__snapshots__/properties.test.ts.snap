// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wikidata Properties Aliases exports as expected 1`] = `
Object {
  "bugTrackerSystem": "P1401",
  "cratesIoId": "P4763",
  "flathubId": "P4655",
  "npmPackage": "P8262",
  "officialWebsite": "P856",
  "publicationDate": "P577",
  "repologyProjectName": "P6931",
  "rubygemsGem": "P5566",
  "softwareVersionIdentifier": "P348",
  "sourceReferenceUrl": "S854",
  "sourceRetrieved": "S813",
}
`;
